nxp: mcxw7x: add config NXP_IEEE802154_MAC required for NBU_MAC

GLegoupil opened the pull request:
For MCXW7X, add new config NXP_IEEE802154_MAC required to use NBU MAC interface
Requires: https://github.com/zephyrproject-rtos/hal_nxp/pull/631

George-Stefan commented on boards/nxp/frdm_mcxw71/frdm_mcxw71_defconfig:
```diff
@@ -10,3 +10,6 @@ CONFIG_SERIAL=y
 CONFIG_UART_CONSOLE=y
 CONFIG_GPIO=y
+
+CONFIG_FPU=y
```
why can't you put this in the sample prj.conf?

mmahadevan108 commented on modules/hal_nxp/mcux/mcux-sdk-ng/components/components.cmake:
```diff
@@ -49,6 +49,15 @@ if(CONFIG_SOC_SERIES_MCXW7XX AND CONFIG_IEEE802154)
 endif()
 
+if(CONFIG_SOC_SERIES_MCXW7XX AND CONFIG_NXP_IEEE802154_MAC)
+  set(CONFIG_MCUX_COMPONENT_driver.spc ON)
```
`CONFIG_DT_HAS_NXP_SPC_ENABLED` should always be enabled for `MCXW7x`. Are you seeing build errors without adding this change?

GLegoupil committed:
drivers: add config NXP_IEEE802154_MAC to pull up MCUX_COMPONENTs for NBU

west: update hal_nxp to integrate mcux: middleware: ieee_802_15_4 MAC: Add
      CONFIG_NXP_IEEE802154_MAC
      zephyr: blobs: add MCXW71 & MCXW72 BLE controller and Host libraries
                     (ieee-802.15.4-mac) to MCUXSDK 25.09.00 release
      This new version of the BLE controller is for ZigBee support and
      implements ieee-802.15.4 MAC interface.

Update connectivity framework for MCXW71 & MCXW72 BLE controller
(ieee-802.15.4 MAC) from MCUXSDK 25.09 release.

Signed-off-by: Guillaume Legoupil <guillaume.legoupil@nxp.com>

mmahadevan108 commented:
> @GLegoupil , please add this information for the new blobs added as part of your HAL PR. Include it in the commit description. [docs.zephyrproject.org/latest/contribute/bin_blobs.html#submission-and-review-process](https://docs.zephyrproject.org/latest/contribute/bin_blobs.html#submission-and-review-process)

Please help add this information and update SHA

GLegoupil commented:
Hi, 
What is the issue failing check? 
[Pull Request/Issue Assigner / Pull Request/Issue Assignment (pull_request_target)](https://github.com/zephyrproject-rtos/zephyr/actions/runs/19528985200/job/55907439530?pr=98790)
scripts/ci/set_assignees.py: AttributeError: 'str' object has no attribute 'get_collaborators_for_path'
=> I don't understand what is going on.
Can you please help ?
